docs/en/engines/table-engines/mergetree-family/invertedindexes.md

```diff
@@ -38,6 +38,7 @@ CREATE TABLE tab
                                 -- Mandatory parameters:
                                 tokenizer = splitByNonAlpha|splitByString(S)|ngrams(N)|array
                                 -- Optional parameters:
+                                [, preprocessor = expression(str)]
```

Comment:
I think we should re-structure this slightly to be consistent with below description:

```sql
                                -- Mandatory parameters:
                                tokenizer = splitByNonAlpha|splitByString(S)|ngrams(N)|array
                                -- Optional parameters:
                                [, preprocessor = expression(str)]
                                -- Optional advanced parameters
                                [, dictionary_block_size = D]
                                [, dictionary_block_frontcoding_compression = B]
                                [, max_cardinality_for_embedded_postings = M]
[, bloom_filter_false_positive_rate = R]
```

Reply:
Do youy mena add subsections like optional and mandatory?

I thought that, but doing so may break the aesthetics of all the document compared with others